libraries/lib-components/EffectInterface.h

```diff
@@ -114,37 +114,37 @@ class COMPONENTS_API EffectDefinitionInterface  /* not final */ : public Compone
    virtual ~EffectDefinitionInterface();
 
    //! Type determines how it behaves.
-   virtual EffectType GetType() = 0;
+   virtual EffectType GetType() const = 0;
 
    //! Determines which menu it appears in; default same as GetType().
-   virtual EffectType GetClassification();
+   virtual EffectType GetClassification() const;
 
    //! Report identifier and user-visible name of the effect protocol
-   virtual EffectFamilySymbol GetFamily() = 0;
+   virtual EffectFamilySymbol GetFamily() const = 0;
 
    //! Whether the effect needs a dialog for entry of settings
-   virtual bool IsInteractive() = 0;
+   virtual bool IsInteractive() const = 0;
 
    //! Whether the effect sorts "above the line" in the menus
-   virtual bool IsDefault() = 0;
+   virtual bool IsDefault() const = 0;
 
    //! Whether the effect supports realtime previewing (while audio is playing).
-   virtual bool SupportsRealtime() = 0;
+   virtual bool SupportsRealtime() const = 0;
 
    //! Whether the effect has any automatable controls.
-   virtual bool SupportsAutomation() = 0;
+   virtual bool SupportsAutomation() const = 0;
```

Comment:
The commit that adds `const` to `SupportsAutomation` has an incorrect commit comment.

Reply:
right, corrected